templates/cloud/public-cloud.html

```diff
@@ -1,170 +1,313 @@
+                    aria-controls="aws-tab"
+                    id="aws">AWS</button>
+            <button class="p-segmented-control__button"
+                    role="tab"
+                    aria-selected="false"
+                    aria-controls="azure-tab"
+                    id="azure"
+                    tabindex="-1">Azure</button>
+            <button class="p-segmented-control__button"
+                    role="tab"
+                    aria-selected="false"
+                    aria-controls="google-cloud-tab"
+                    id="google-cloud"
+                    tabindex="-1">Google Cloud</button>
+            <button class="p-segmented-control__button"
+                    role="tab"
+                    aria-selected="false"
+                    aria-controls="ibm-cloud-tab"
+                    id="ibm-cloud"
+                    tabindex="-1">IBM Cloud</button>
+            <button class="p-segmented-control__button"
+                    role="tab"
+                    aria-selected="false"
+                    aria-controls="oracle-cloud-tab"
+                    id="oracle-cloud"
+                    tabindex="-1">Oracle Cloud</button>
+          </div>
+        </div>
+        <form class="u-hide--large u-hide--medium">
+          <select name="boardSelect" id="boardSelect">
+            <option value="aws-tab" selected="">AWS</option>
+            <option value="azure-tab">Azure</option>
+            <option value="google-cloud-tab">Google Cloud</option>
+            <option value="ibm-cloud-tab">IBM</option>
+            <option value="oracle-cloud-tab">Oracle</option>
+          </select>
+        </form>
       </div>
-      <p>Juju is the fastest way to deploy and scale out your workloads on the leading public clouds. With Juju charm bundles, you can launch an entire cloud environment with one click, or just one command using the CLI. Use Juju to  deploy your entire application infrastructure, all in one.</p>
-      <p><a href="https://jujucharms.com/">Learn more about Juju&nbsp;&rsaquo;</a></p>
     </div>
-    <div class="col-6 p-divider__block">
-      <h2 class="p-heading--3">Trusted Container (OCI) Images</h2>
-      <div class="u-hide--medium u-hide--small u-align--center u-sv1">
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/fcb97fce-canonical-on-amazon-ecr.png",
-            alt="Canonical on Amazon screenshot",
-            height="208",
-            width="371",
-            hi_def=True,
-            loading="lazy",
-          ) | safe
-        }}
+    <div class="row">
+      <div class="col-start-large-4 col-9 col-medium-6">
+        <hr class="is-muted" />
+        {% include "download/cloud/tabs/_aws.html" %}
+        {% include "download/cloud/tabs/_azure.html" %}
+        {% include "download/cloud/tabs/_google-cloud.html" %}
+        {% include "download/cloud/tabs/_ibm-cloud.html" %}
+        {% include "download/cloud/tabs/_oracle-cloud.html" %}
```

Comment:
Please take a look at the [copydoc](https://docs.google.com/document/d/1kp7QQXst1EbCjZjzWZP3YZwfmG0EIXLfQoRUVaMEZng/edit), there are links missing from these two sections